compiler/src/dmd/common/unicode_tables.d

```diff
@@ -0,0 +1,2424 @@
+    [0x1EE3B, 0x1EE3B],
+    [0x1EE42, 0x1EE42],
+    [0x1EE47, 0x1EE47],
+    [0x1EE49, 0x1EE49],
+    [0x1EE4B, 0x1EE4B],
+    [0x1EE4D, 0x1EE4F],
+    [0x1EE51, 0x1EE52],
+    [0x1EE54, 0x1EE54],
+    [0x1EE57, 0x1EE57],
+    [0x1EE59, 0x1EE59],
+    [0x1EE5B, 0x1EE5B],
+    [0x1EE5D, 0x1EE5D],
+    [0x1EE5F, 0x1EE5F],
+    [0x1EE61, 0x1EE62],
+    [0x1EE64, 0x1EE64],
+    [0x1EE67, 0x1EE6A],
+    [0x1EE6C, 0x1EE72],
+    [0x1EE74, 0x1EE77],
+    [0x1EE79, 0x1EE7C],
+    [0x1EE7E, 0x1EE7E],
+    [0x1EE80, 0x1EE89],
+    [0x1EE8B, 0x1EE9B],
+    [0x1EEA1, 0x1EEA3],
+    [0x1EEA5, 0x1EEA9],
+    [0x1EEAB, 0x1EEBB],
+    [0x20000, 0x2A6DF],
+    [0x2A700, 0x2B739],
+    [0x2B740, 0x2B81D],
+    [0x2B820, 0x2CEA1],
+    [0x2CEB0, 0x2EBE0],
+    [0x2F800, 0x2FA1D],
+    [0x30000, 0x3134A],
+    [0x31350, 0x323AF],
+];
+/**
+UAX31 profile Start: Legacy{without XID_Start}
+Entries: 439
+*/
+static immutable dchar[2][] Start_Legacy_NonXID_Start = [
+    [0xB7, 0xB7],
+    [0x37A, 0x37A],
+    [0x5B0, 0x5B9],
+    [0x5BB, 0x5BD],
+    [0x5BF, 0x5BF],
+    [0x5C1, 0x5C2],
+    [0x64B, 0x652],
+    [0x660, 0x669],
+    [0x670, 0x670],
+    [0x6D4, 0x6D4],
+    [0x6D6, 0x6DC],
+    [0x6E7, 0x6E8],
+    [0x6EA, 0x6ED],
+    [0x6F0, 0x6F9],
+    [0x901, 0x903],
+    [0x93E, 0x94D],
+    [0x951, 0x952],
+    [0x962, 0x963],
+    [0x966, 0x96F],
+    [0x981, 0x983],
+    [0x9BE, 0x9C4],
+    [0x9C7, 0x9C8],
+    [0x9CB, 0x9CD],
+    [0x9E2, 0x9E3],
+    [0x9E6, 0x9EF],
+    [0xA02, 0xA02],
+    [0xA3E, 0xA42],
+    [0xA47, 0xA48],
+    [0xA4B, 0xA4D],
+    [0xA66, 0xA6F],
+    [0xA81, 0xA83],
+    [0xABE, 0xAC5],
+    [0xAC7, 0xAC9],
+    [0xACB, 0xACD],
+    [0xAE6, 0xAEF],
+    [0xB01, 0xB03],
+    [0xB3E, 0xB43],
+    [0xB47, 0xB48],
+    [0xB4B, 0xB4D],
+    [0xB66, 0xB6F],
+    [0xB82, 0xB82],
+    [0xBBE, 0xBC2],
+    [0xBC6, 0xBC8],
+    [0xBCA, 0xBCD],
+    [0xBE7, 0xBEF],
+    [0xC01, 0xC03],
+    [0xC3E, 0xC44],
+    [0xC46, 0xC48],
+    [0xC4A, 0xC4D],
+    [0xC66, 0xC6F],
+    [0xC82, 0xC83],
+    [0xCBE, 0xCC4],
+    [0xCC6, 0xCC8],
+    [0xCCA, 0xCCD],
+    [0xCE6, 0xCEF],
+    [0xD02, 0xD03],
+    [0xD3E, 0xD43],
+    [0xD46, 0xD48],
+    [0xD4A, 0xD4D],
+    [0xD66, 0xD6F],
+    [0xE31, 0xE31],
+    [0xE33, 0xE3A],
+    [0xE47, 0xE5B],
+    [0xEB1, 0xEB1],
+    [0xEB3, 0xEB9],
+    [0xEBB, 0xEBC],
+    [0xEC8, 0xECD],
+    [0xED0, 0xED9],
+    [0xF18, 0xF19],
+    [0xF20, 0xF33],
+    [0xF35, 0xF35],
+    [0xF37, 0xF37],
+    [0xF39, 0xF39],
+    [0xF3E, 0xF3F],
+    [0xF71, 0xF84],
+    [0xF86, 0xF87],
+    [0xF90, 0xF95],
+    [0xF97, 0xF97],
+    [0xF99, 0xFAD],
+    [0xFB1, 0xFB7],
+    [0xFB9, 0xFB9],
+    [0x203F, 0x2040],
+    [0x309B, 0x309C],
+    [0x30FB, 0x30FB],
+];
+/// UAX31 profile Continue: XID_Continue{without legacy}{without non-starters}
+static immutable dchar[2][] Continue_Starter = [
+    [0x30, 0x39],
+    [0x41, 0x5A],
+    [0x5F, 0x5F],
+    [0x61, 0x7A],
+    [0xAA, 0xAA],
+    [0xB5, 0xB5],
+    [0xB7, 0xB7],
```

Comment:
middle dot?

Reply:
Yeah, thats a legacy table, which is what we already have.

Can't break people's build with this PR, gotta do deprecations.

Reply:
Oh wait that's in non-legacy, I will need to verify that it should be there.

Reply:
Okay fixed.

Reply:
Nope I was right, if its in XID_Continue it should be there doh.